buildSrc/src/main/groovy/neoforge.formatting-conventions.gradle

```diff
@@ -55,28 +59,27 @@ spotless {
         custom 'jetbrainsNullable', { String fileContents ->
             fileContents.replace('javax.annotation.Nullable', 'org.jetbrains.annotations.Nullable')
         }
-        bumpThisNumberIfACustomStepChanges(3)
```

Comment:
Could you explain why this line was removed? Particularly, how Immaculate differs from Spotless in how it handles custom steps and their changes.

Reply:
All caching goes through normal up-to-date checking -- a change to the custom step implementation will make the up-to-date check fail and thus, all this is handled automatically.

Reply:
Following up on this further: immaculate just uses normal old up-to-date checking _everywhere_ combined with incremental file collections to not reprocess every single file when one changes. This means, basically, that you shouldn't get the occasional messed up caches issue like spotless could have, and that you don't need any custom logic like spotless needed for custom steps to have functional up-to-date checking, as the implementation is part of the key.